modules/costs/app/components/time_entries/entry_dialog_component.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render(Primer::Alpha::Dialog.new(title: t('caption_log_time_dialog'), size: :large, id: MODAL_ID, data: { "controller" => "time-entry",  "application-target" => "dynamic", "time-entry-id" => time_entry.id, "ongoing" => time_entry.ongoing? })) do |d| %>
```

Comment:
```suggestion
<%= render(Primer::Alpha::Dialog.new(title: t('caption_log_time_dialog'), size: :medium_portrait, id: MODAL_ID, data: { "controller" => "time-entry",  "application-target" => "dynamic", "time-entry-id" => time_entry.id, "ongoing" => time_entry.ongoing? })) do |d| %>
```

I think this would look better in :medium_portrait.